JumpscaleCore/clients/threebot/README.md

```diff
@@ -0,0 +1,51 @@
+# 3Bot client
+
+## Example on how to use the threebot client
+
+- we need threebot server running and a package to test with
+
+```python
+cl = j.servers.threebot.local_start_default(web=True)
+```
+
+- Add a test package and reload the server after loading the new package
+
+```python
+cl.actors.package_manager.package_add(path="/sandbox/code/github/threefoldtech/jumpscaleX_threebot/ThreeBotPackages/tfgrid/registry")
+cl.reload()
+```
+
+- Create the `default` instance from threebot.me
+
+```python
+ j.tools.threebot.init_my_threebot(myidentity='default', name='hamdy farag', email='ham1dy@d.com', description=None, ipaddr='127.0.0.1', interactive=False)```
+```
+
+- Get the client and use it (registering my threebot.me id), It will authenticate automatically
+
+```python
+c = j.clients.threebot.client_get(threebot=1)
```

Comment:
Can you change this to `j.tools.threebot.me.default.id` instead of 1

Reply:
done